.github/workflows/R-CMD-check-devel.yaml

```diff
@@ -0,0 +1,75 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [devel]
```

Comment:
@AARON-CLARK I think we might want to remove these two lines. Since `devel` is designed to always be an open PR for `master`. I don't have strong feelings about it though if we want to trigger both jobs though.

Reply:
After some discussion, we decided to leave this as-is. There are (rare) occasions where there may be a rogue commit to `devel` outside of a PR, and it'd be nice to know if the r-cmd-check using `renv.lock` worked or not.